src/Raygun.NetCore.Blazor.WebAssembly/Extensions/WebAssemblyHostBuilderExtensions.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.AspNetCore.Components.WebAssembly.Hosting
     /// <summary>
     /// Extensions for registering RaygunBlazorClient and related services with a Blazor WebAssembly application.
     /// </summary>
-    public static class Raygun_WebAssembly_WebAssemblyHostBuilderExtensions
+    public static class RaygunWebAssemblyWebAssemblyHostBuilderExtensions
```

Comment:
Perhaps outside the scope of this PR, but the double up on "WebAssembly" seems unnecessary

Reply:
Just so you folks are aware, the underscores here were to separate logical concepts for these extensions. There are some edge cases where people end up having to reference the class name itself, either through reflection or other means, to call the extension method. It doesn't happen often, but it happens. I started using the previous pattern so that if Reflection is somehow required, you can distinguish this class from the dozens of other `IServiceCollectionExtensions`-type classes floating out there.

In this case, there conceivably might be other WebAssemblyHostBuilderExtensions in the Raygun WebAssembly code (I was thinking about the Offline plugin I had been planning) so that's why "WebAssembly" had been doubled-up.

HTH!

Reply:
Thank you! I did not realize the IDE autoformatting removed the underscores, I'll see what I can do about this.

Reply:
I have reverted this change, we can reapply later on if we decide to.

Reply:
Yep, can revisit this later. I prefer the names not including the underscores. The middle section of all those names may not be necessary - unless as Robert mentions there could be multiple extension classes and hence a way to give them different names. Until then though, I think it would be good to simplify the names to RaygunNameOfExtendedClassExtensions.